src/classes/content/wordpress/class-abstract-wordpress-content-service.php

```diff
@@ -69,13 +70,24 @@ public function get_about_jsonld( $content_id ) {
 
 	/**
 	 * @param Wordpress_Content_Id $content_id
-	 * @param string               $value
+	 * @param string $value
+	 *
+	 * @throws Exception If the 'match_name' column does not exist in the database table.
 	 */
 	public function set_about_jsonld( $content_id, $value ) {
 		global $wpdb;
 
 		// Cleanup value.
-		$value = ( is_string( $value ) && strlen( $value ) > 2 ) ? $value : null;
+		$value      = ( is_string( $value ) && strlen( $value ) > 2 ) ? $value : null;
+		$match_name = "NULL";
+
+		if ( $value ) {
+			// Check if the 'match_name' column exists in the database table
+			$columns = $wpdb->get_col_info( 'name', 0 );
```

Comment:
Do we need this check? The plugin upgrade should add this column, so that this check should be irrelevant?